app/validation_test.go

```diff
@@ -133,3 +133,138 @@ func TestValidationHash(t *testing.T) {
 	}
 	assert.Len(t, errs, len(allExpected))
 }
+
+func TestMandatoryDescriptions(t *testing.T) {
```

Comment:
Is this `t.Parallel()`able?

Same with the other new tests. (The old ones too, really, but we don't care about those right now)

Reply:
Should be, yeah